src/eu/packages/eu-component-button/button.story.js

```diff
@@ -0,0 +1,61 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
```

Comment:
So, this is a common pattern, i see. We have two systems, EC and EU and for how "similar" they might look, they are independent and we should avoid confusion.
Here we are only dealing with a demo of the component, in the end, the twig component per se is not affected by this, but still we want to keep things separated, so please use only resources from the right ecosystem, in this case EU ;)

Reply:
Fixed in latest commit [4a6eb06](https://github.com/ec-europa/ecl-twig/pull/14/commits/4a6eb062c24d03bb9dc0a220fe8ec66ae6f01d4d)